src/modules/reviews.js

```diff
@@ -94,13 +95,31 @@ class Reviews {
         const banner = document.querySelector('.reviews__banner')
         banner.insertAdjacentElement('beforeend', this.getStars(this.summaryData.rating.rating, 21,'mb-1 px-2'))
         banner.innerHTML += '<img class="d-none d-md-inline" srcset="https://res.cloudinary.com/dy7hqiitw/image/upload/w_60/worktop-express-uk/brands/feefo-logo.png 1x, https://res.cloudinary.com/dy7hqiitw/image/upload/w_120/worktop-express-uk/brands/feefo-logo.png 2x" alt="Feefo logo">';
+        
+        switch(this.merchantID) {
+            case 'worktop-express-gmbh': // WEX DE
+                this.locations.summary.innerHTML += `${this.getStars(this.summaryData.rating.rating, 27, null, '<img class="d-inline-block pl-2" srcset="https://res.cloudinary.com/dy7hqiitw/image/upload/w_60/worktop-express-uk/brands/feefo-logo.png 1x, https://res.cloudinary.com/dy7hqiitw/image/upload/w_120/worktop-express-uk/brands/feefo-logo.png 2x" alt="Feefo logo">').outerHTML}`;
+                this.locations.summary.innerHTML += `<div class="font-14 colour-grey4">Bewertet mit ${this.summaryData.rating.rating} / 5 basierend auf ${this.summaryData.rating.product.count} Bewertungen</div>`;
+                break;
 
-        this.locations.summary.innerHTML += `${this.getStars(this.summaryData.rating.rating, 27, null, '<img class="d-inline-block pl-2" srcset="https://res.cloudinary.com/dy7hqiitw/image/upload/w_60/worktop-express-uk/brands/feefo-logo.png 1x, https://res.cloudinary.com/dy7hqiitw/image/upload/w_120/worktop-express-uk/brands/feefo-logo.png 2x" alt="Feefo logo">').outerHTML}`
-        this.locations.summary.innerHTML += `<div class="font-14 colour-grey4">Rated ${this.summaryData.rating.rating} / 5 Based on ${this.summaryData.rating.product.count} reviews</div>`
+            default: // WEX UK
+                this.locations.summary.innerHTML += `${this.getStars(this.summaryData.rating.rating, 27, null, '<img class="d-inline-block pl-2" srcset="https://res.cloudinary.com/dy7hqiitw/image/upload/w_60/worktop-express-uk/brands/feefo-logo.png 1x, https://res.cloudinary.com/dy7hqiitw/image/upload/w_120/worktop-express-uk/brands/feefo-logo.png 2x" alt="Feefo logo">').outerHTML}`;
+                this.locations.summary.innerHTML += `<div class="font-14 colour-grey4">Rated ${this.summaryData.rating.rating} / 5 Based on ${this.summaryData.rating.product.count} reviews</div>`;
+        }
     }
     setHead = () => {
         if (this.summaryData.rating.rating < 3) return
-        const headString = `<div class="reviews-head__index col-md-auto col-md px-0 font-12 font-md-15 underline colour-grey3 pt-1 pt-md-0 pl-md-2">(${this.summaryData.rating.product.count} customer reviews)</div>`;
```

Comment:
Changed from "const" to "let", as below switch statement will update this variable based on the this.merchantID